jekyll/_cci2/deployments-and-uploads.md

```diff
@@ -1,8 +0,0 @@
----
```

Comment:
Rather than delete files, please add the `redirect` front-matter directive (eg value: `/docs/2.0/deployments/`) to avoid 404s.